src/test/regress/expected/citus_cdc_basic_distributed_table.out

```diff
@@ -0,0 +1,193 @@
+        PUBLICATION cdc_publication
+               WITH (
+                   create_slot=false,
+                   enabled=true,
+                   slot_name='cdc_replication_slot',
+                   copy_data=false);
+-- Create subscriptions to receive events from worker nodes for the distributed table.
+CREATE SUBSCRIPTION cdc_subscription_2
+        CONNECTION 'host=localhost port=xxxxx user=postgres dbname=regression'
+        PUBLICATION cdc_publication
+               WITH (
+                   create_slot=false,
+                   enabled=true,
+                   slot_name='cdc_replication_slot',
+                   copy_data=false);
+-- END 2
+-- BEGIN 3: connect to co-ordinator node and populate the distributed table and store the current LSN values of worker nodes in variables.
+\c - postgres - :master_port
+SET search_path TO "citus_cdc_test_schema";
+INSERT INTO sensors
+	SELECT i, '2020-01-05', '{}', 11011.10, 'A', 'I <3 Citus'
+	FROM generate_series(0,1000)i;
+SELECT COUNT(*) AS distribued_sensor_count FROM sensors \gset
+\c - postgres - :worker_1_port
+SELECT pg_current_wal_lsn()-'0/0' AS remote_lsn_1 \gset
+\c - postgres - :worker_2_port
+SELECT pg_current_wal_lsn()-'0/0' AS remote_lsn_2 \gset
+-- END 3: connect to co-ordinator node and populate the distributed table and note down the LSN values of worker nodes in variables.
+-- BEGIN 4: connect to CDC client node and wait for the subscriptions to catch up with the worker nodes.
+\c - postgres - :worker_3_port
+-- Create subscription to connect to co-ordinator and replicate the sensors table.
+SET search_path TO "citus_cdc_test_schema";
+-- This function waits for the subscription to catch up with the given remote node's LSN.
+CREATE OR REPLACE FUNCTION citus_cdc_test_schema.wait_for_subscription_to_catchup(subscription_name text, remoteLsn bigint) RETURNS void AS $$
+DECLARE
+	receivedLsn bigint;
+BEGIN
+    EXECUTE FORMAT('SELECT min(latest_end_lsn)-''0/0'' FROM pg_stat_subscription WHERE subname = ''%s'';', subscription_name) INTO receivedLsn;
+	WHILE receivedLsn IS NULL  OR remoteLsn > receivedLsn LOOP
+		EXECUTE FORMAT('SELECT min(latest_end_lsn)-''0/0'' FROM pg_stat_subscription WHERE subname = ''%s'';', subscription_name) INTO receivedLsn;
+		PERFORM pg_sleep(1);
+    END LOOP;
+END$$ LANGUAGE plpgsql;
+SELECT wait_for_subscription_to_catchup('cdc_subscription_1',:remote_lsn_1);
+ wait_for_subscription_to_catchup
+---------------------------------------------------------------------
+
+(1 row)
+
+SELECT wait_for_subscription_to_catchup('cdc_subscription_2',:remote_lsn_2);
+ wait_for_subscription_to_catchup
+---------------------------------------------------------------------
+
+(1 row)
+
+SELECT COUNT(*) AS cdc_client_sensor_count FROM sensors \gset
+\echo "expected_sensor_count: " :distribued_sensor_count "cdc_client_sensor_count: " :cdc_client_sensor_count
+"expected_sensor_count: " 1001 "cdc_client_sensor_count: " 1001

```

Comment:
logical replication from Citus to Postgres! super cool :)

Reply:
Yes :) I changed this to compare the entrie distributed table exported to a CSV file with the replicated table in CDC client.
So if there are any mismatch in replicated data, it should be caught by the comparison.